lib/std/os/uefi/protocols/device_path_protocol.zig

```diff
@@ -363,14 +363,14 @@ pub const MessagingDevicePath = union(Subtype) {
         }
     };
 
-    pub const DeviceLogicalUnitDevicePath = packed struct {
+    pub const DeviceLogicalUnitDevicePath = extern struct {
         type: DevicePathType,
         subtype: Subtype,
         length: u16,
         lun: u8,
     };
```

Comment:
With this change, there is a difference with these structs compared to before.

The prior `packed` version would not have had padding but these extern ones do, unless the alignment of certain fields is forced to be smaller.

```zig
// @sizeOf() == 6
pub const DeviceLogicalUnitDevicePath = extern struct {
    type: DevicePathType,
    subtype: Subtype,
    length: u16,
    lun: u8,
};

// @sizeOf() == 5
pub const DeviceLogicalUnitDevicePathWithAlignment = extern struct {
    type: DevicePathType,
    subtype: Subtype,
    length: u16 align(1),
    lun: u8,
};
```
`DeviceLogicalUnitDevicePath` was chosen as an example, this affects other structs too.